src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -54,6 +59,7 @@ protected SecorConfig initialValue() {
                 properties.setProperty(entry.getKey().toString(), entry.getValue());
             }
 
+            LOG.debug("Configuration: " + ConfigurationUtils.toString(properties));
```

Comment:
This '+' operation might be expensive, does this scheme work?

LOG.debug("Configuration: {}", properties)

If not, you can do:
    if (LOG.isDebugEnabled()) {
        LOG.debug ...
    }

Reply:
Good point. I'll try, and update the branch then.